controllers/feeds/getReaction.js

```diff
@@ -34,7 +37,8 @@ module.exports = async (req, res) => {
               req,
               detailFeed.actor.id,
               myAnonymUser,
-              dataUser.user_id
+              dataUser.user_id,
+              karmaScores
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes look good, but it's important to ensure that the `getReaction` function can handle the new `karmaScores` argument correctly. If the function doesn't expect this argument or doesn't know how to handle it, this could lead to unexpected behavior. Please make sure that the `getReaction` function has been updated accordingly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->